primitives/evm-trace/events/src/gasometer.rs

```diff
@@ -30,7 +30,6 @@ impl Snapshot {
 	}
 }
 
-#[cfg(feature = "evm-tracing")]
```

Comment:
Is this expected?

Remove or just un-comment commented?

Reply:
We can remove it. I have tested the tracing process locally. It works well.

Reply:
Just curious, any reason for making this change since enabling this makes the compiler compile less?

Reply:
There may be a slight impact on the size of the binary, the reason for doing this is to be consistent with upstream. I am discussing talks with the moonbeam developers about the possibility of extracting the EVM tracing-related libraries into a separate library so that we can rely on them directly and reduce maintenance costs.